pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -86,6 +121,9 @@ type stateData struct {
 
 	// nodeAllocations caches the result of Filter for the nodes.
 	nodeAllocations map[string][]resourceapi.AllocationResult
+	// nodeClaim caches the special claim for extended resource backed by DRA
+	// created during Filter for the nodes
+	nodeExtendedResourceClaim map[string]*resourceapi.ResourceClaim
```

Comment:
What is the map key? Probably the node name - please document that.

Reply:
Yes, the key is node name. added that to the comment.

Reply:
I think it's better to change `nodeAllocations` into a `map[string]nodeAllocation` with
```
type nodeAllocation struct {
    allocationResults []resourceapi.AllocationResult
    extendedResourceClaim *resourceapi.ResourceClaim
}
```